### PR TITLE
 feat(Cards): Added new Card components and updated style of Cards

### PR DESCRIPTION
**Pull request by HHogg**
[>>> Netlify Link <<<](https://rename-cards--hhogg-axiom.netlify.com/)

### CODE REVIEW

File: src/components/card/Card.js [hunk 1]
```diff
@@ -2,26 +2,81 @@ import PropTypes from 'prop-types';
...
+import './Card.css';
+
+const cardListStyleProps = {
+  seamless: {
+    border: false,
+    color: 'default',
+    elevation: 'x0',
+    space: 'x0',
+  },
+  separate :{
```
Comment (thatguynamedandy) [inline src/components/card/Card.js]: OCD `seamless: {` and `separate :{`

File: src/components/card/Card.js [hunk 2]
```diff
@@ -2,26 +2,81 @@ import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import classnames from 'classnames';
 import Base from '../base/Base';
+import './Card.css';
+
+const cardListStyleProps = {
+  seamless: {
+    border: false,
+    color: 'default',
```
Comment (thatguynamedandy) [inline src/components/card/Card.js]: Minor again but there maybe isn't a need to specify `color` here as it's always the same.

File: src/components/card/Card.js [hunk 3]
```diff
@@ -2,26 +2,81 @@ import PropTypes from 'prop-types';
...
+  };
+
+  static defaultProps = {
+    color: 'default',
+    elevation: 'x0',
+    size: 'medium',
+  };
+
+  static contextTypes = {
+    cardListStyle: PropTypes.string,
```
Comment (thatguynamedandy) [inline src/components/card/Card.js]: can this be `oneOf(['seamless', 'separate'])`?

File: src/components/card/Card.js (hunk 2 above)
Comment (HHogg) [reply src/components/card/Card.js]: This is to enforce the style so it can't be overridden when a Card is used within a list.

**Pull request merged**